composer.json

```diff
@@ -19,13 +19,12 @@
     ],
     "require": {
         "php": ">=7.0",
-        "illuminate/support": ">=5.0,<5.7",
-        "illuminate/console": ">=5.0,<5.7",
-        "illuminate/validation": ">=5.0,<5.7"
+        "laravel/framework": ">=5.0.0"
```

Comment:
What's the reasoning behind this change?

Reply:
The `orchestra/testbench` package used to run tests in a Laravel environment appeared to take issue with the requirements.

After double checking, I can't seem to replicate the issue. I'll go ahead and update the requirements accordingly. Thanks for bringing that up.